src/Annotation/Groups.php

```diff
@@ -11,19 +11,25 @@
 #[\Attribute(\Attribute::TARGET_METHOD | \Attribute::TARGET_PROPERTY)]
 final class Groups
 {
+    use AnnotationUtilsTrait;
+
     /** @var array<string> @Required */
     public $groups = [];
 
     public function __construct(array $values = [], array $groups = [])
     {
-        if (array_key_exists('value', $values)) {
-            $groups = $values['value'];
-        }
-
-        if (array_key_exists('groups', $values)) {
-            $groups = $values['groups'];
+        $vars = get_defined_vars();
+        /*
```

Comment:
TBH I don't like to have a different behaviour depending on the annotation, but I don't see any other way :)

Reply:
yea...